.github/actions/install-risc0/action.yml

```diff
@@ -0,0 +1,67 @@
+name: "Install Risc0"
```

Comment:
It seems that this action is not used in the `workflows/packaging.yml`? Could you teach me where this action is executed from?

Reply:
This one is called from the `install-dependencies` action. Do you think it's a better idea to have it as a separate step in the main workflow?

Reply:
Ah, I missed it, sorry :) No, the current workflow is good in my opinion.